ztp/resource-generator/src/get-pre-sync-rv.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Log in and setup environment
+. $(dirname "$0")/common.sh $1
```

Comment:
If this is the way it's used, perhaps it's better to put the code in a function, as opposed to the work being done as a side-effect of sourcing it?

```
. $(dirname "$0")/common.sh

# Login and setup environment
waitForLogin $1
```

Reply:
This approach would also lend itself to eventually adding a unit test around the `waitForLogin` function or whatever you want to call it.

Reply:
Agreed. Reworked to be a function. Side-effects from loading aren't nice :)